docs/website/package-lock.json

```diff
@@ -2236,9 +2236,9 @@
       }
     },
     "node_modules/chokidar-cli/node_modules/ansi-regex": {
-      "version": "4.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-4.1.0.tgz",
-      "integrity": "sha512-1apePfXM1UOSqw0o9IiFAovVz9M5S1Dg+4TrDwfMewQ6p/rmMueb7tWZjQ1rx4Loy1ArBggoqGpfqqdI4rondg==",
+      "version": "4.1.1",
```

Comment:
Do we need update our package-lock.json?

Reply:
there was a high vulnerability message so I ran the command `npm audit fix` and it fixed it) so I guess yes